study-builder/studies/angio/about-you.conf

```diff
@@ -14,16 +14,16 @@
   "translatedNames": [
     {
       "language": "en",
-      "text": "Join the movement: tell us about yourself"
+      "text": "Initial Enrollment Survey"
     }
   ],
-  "translatedSubtitles": [],
-  "translatedDashboardNames": [
+  "translatedTitles": [
```

Comment:
@charlottedye you'll need to update your Prion config files, similar to what I have done here for the other studies.